misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/BTestRunner.java

```diff
@@ -264,9 +292,9 @@ private void execute(boolean buildWithTests) {
                         } catch (Throwable e) {
                             shouldSkipTest.set(true);
                             errorMsg = String.format("\t[fail] " + beforeEachTest.getName() +
-                                                     " [before each test function for the test %s] :\n\t    %s",
-                                                     test.getTestFunction().getName(),
-                                                     Utils.formatError(e.getMessage()));
+                                            " [before each test function for the test %s] :\n\t    %s",
+                                    test.getTestFunction().getName(),
+                                    Utils.formatError(e.getMessage()));
```

Comment:
"Utils.java" in Testerina was renamed to "TesterinaUtils.java".  So shall we change the usages of 'Utils' to 'TesterinaUtils?'

Reply:
done